cove/cove_360/views.py

```diff
@@ -91,6 +91,7 @@ def explore_360(request, pk, template='cove_360/explore.html'):
     upload_url = db_data.upload_url()
     file_name = db_data.original_file.file.name
     file_type = context['file_type']
+    schema_360 = Schema360(upload_dir)
```

Comment:
Is it OK for the upload dir to be a working dir, can there be any name conflicts etc?

Reply:
That's a good point, if someone were to upload a file with the same name as the schema or extension file. The upload_dir itself is fine as it's using django's builtin mecahnism for unique dirs, however I think putting the schema stuff in a directory would be an improvement.